src/app/features/search/multiple-search-results/multiple-search-results.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Location } from '@angular/common';
+import { Component, OnDestroy } from '@angular/core';
+import { GlobalErrorService } from '@core/components/global-error/global-error.service';
+import { VehicleTechRecordModel } from '@models/vehicle-tech-record.model';
+import { select, Store } from '@ngrx/store';
+import { SEARCH_TYPES, TechnicalRecordService } from '@services/technical-record/technical-record.service';
+import { selectQueryParams } from '@store/router/selectors/router.selectors';
+import { Observable, Subject, takeUntil } from 'rxjs';
+
+@Component({
+  selector: 'app-multiple-search-results',
+  templateUrl: './multiple-search-results.component.html',
+  styleUrls: ['multiple-search-results.component.scss']
+})
+export class MultipleSearchResultsComponent implements OnDestroy {
+  vehicleTechRecords$: Observable<VehicleTechRecordModel[]>;
+  ngDestroy$ = new Subject();
+
+  constructor(public globalErrorService: GlobalErrorService, private technicalRecordService: TechnicalRecordService, private store: Store, private location: Location) {
+    this.store //
+      .pipe(select(selectQueryParams), takeUntil(this.ngDestroy$))
+      .subscribe((params) => {
+        if (Object.keys(params).length === 1) {
+          const type = Object.keys(params)[0] as SEARCH_TYPES;
+          const searchTerm = params[type] as string;
+
+          if (searchTerm && Object.values(SEARCH_TYPES).includes(type as SEARCH_TYPES)) {
+            this.globalErrorService.clearError();
+            this.technicalRecordService.searchBy({ type, searchTerm });
+          }
+        }
+      });
+
+    this.vehicleTechRecords$ = this.technicalRecordService.vehicleTechRecords$;
+  }
+
+  navigateBack() {
```

Comment:
Maybe a reusable function?

Reply:
What do you have in mind? My thought is that, in the near future when multiple pages require a back button, we could create a base component in the core module which all these pages would extend.

But this should be its own ticket and PR.

Reply:
Not sure if relevant but there was something on our side where we were deliberately not putting back buttons on pages, unsure if this is something that we're just doing by accident or if there's a reason for it?

Reply:
I am resolving this due to inactivity, but do take it with me offline if you want to create some base shared components and functions as I'm all about that.